Neos.Flow/Classes/ObjectManagement/DependencyInjection/ProxyClassBuilder.php

```diff
@@ -466,7 +468,33 @@ public function buildPropertyInjectionCodeByString(Configuration $objectConfigur
             return $result;
         }
 
-        if ($propertyConfiguration->isLazyLoading() && $this->objectConfigurations[$propertyObjectName]->getScope() !== Configuration::SCOPE_PROTOTYPE) {
+        $propertyClassHasInterfaceWithConstructor = false;
+        $propertyClassHasFinalMethod = false;
+        if ($propertyClassName) {
+            foreach ((new ClassReflection($propertyClassName))->getInterfaceNames() as $interfaceName) {
+                if (method_exists($interfaceName, '__construct')) {
+                    $propertyClassHasInterfaceWithConstructor = true;
+                    break;
+                }
+            }
+            foreach (get_class_methods($propertyClassName) as $methodName) {
+                if ($this->reflectionService->isMethodFinal($propertyClassName, $methodName)) {
+                    $propertyClassHasFinalMethod = true;
```

Comment:
```suggestion
                    $propertyClassHasFinalMethod = true;
                    break;
```
microoptimization

Reply:
we are basically rewriting js `Array.some()` - and i just looked it up - there is no php `array_some` lol

Reply:
... this was not added ...

Reply:
> ... this was not added ...

I don't get it. What was not added where?

Reply:
The suggestion was not committed to this PR. But now it was.

Reply:
Yes, got it. And now was wondering why I can't find the comment, but it was already resolved. 🤣